pandas/core/indexes/base.py

```diff
@@ -2005,15 +2005,35 @@ def __contains__(self, key):
             return False
 
     _index_shared_docs['contains'] = """
-        return a boolean if this key is IN the index
+        Return a boolean if this key is in the index.
```

Comment:
Minor thing, and I know this is the original description and not yours, but this sounds like it a boolean is returned only if the key is in the index. Not that a boolean is always returned, on whether the key is in the index. Do you mind rephrasing it to be more correct please?

Reply:
Sure, I'll rephrase it in the next commit as a boolean indicator. Apparently, there are few more docstrings with similar phrasing like [`Series.is_unique`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.is_unique.html#pandas.Series.is_unique), [`Series.is_monotonic`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.is_monotonic.html#pandas.Series.is_monotonic), [`Series.is_monotonic_increasing`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.is_monotonic_increasing.html#pandas.Series.is_monotonic_increasing), [`Series.is_monotonic_decreasing`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.is_monotonic_decreasing.html#pandas.Series.is_monotonic_decreasing). These too can be corrected in future if seems appropriate.